corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java

```diff
@@ -516,52 +519,84 @@ public CorfuDynamicRecord editRecord(String namespace, String tableName,
         return null;
     }
 
+    /**
+     * Delete all the records from a file record in a table and namespace
+     * @param namespace namespace of the table
+     * @param tableName name of the table
+     * @param pathToKeysFile path to file having each key in a json line
+     * @param batchSize path to file having each key in a json line
+     * @return number of keys deleted.
+     */
+    public int deleteRecordsFromFile(String namespace, String tableName, String pathToKeysFile, int batchSize) {
+        List<String> recordsToDelete = Collections.emptyList();
+        System.out.println("\n======================\n");
+        System.out.println("\nReading all json keys to be deleted from "+pathToKeysFile+"\n");
+        try {
+            recordsToDelete = Files.readAllLines(Paths.get(pathToKeysFile), StandardCharsets.UTF_8);
+        } catch (IOException e) {
+            log.error("Unable to read file {}"+pathToKeysFile, e);
+            return 0;
+        }
+        return deleteRecords(namespace, tableName, recordsToDelete, batchSize);
+    }
+
     /**
      * Delete a record in a table and namespace
      * @param namespace namespace of the table
      * @param tableName name of the table
-     * @param keyToDelete JSON string representing the key protobuf that needs to be deleted.
+     * @param keysToDelete array of JSON strings of key protobuf that needs to be deleted.
+     * @param batchSize number of records to group into a single corfu transaction
      * @return number of keys deleted.
      */
     @SuppressWarnings("checkstyle:magicnumber")
-    public int deleteRecord(String namespace, String tableName, String keyToDelete) {
+    public int deleteRecords(String namespace, String tableName, List<String> keysToDelete, int batchSize) {
```

Comment:
how will this command look from the corfu tool runner script in the client?  We had received feedback to keep it easy to type and less error prone.  How does passing the whole list translate in the command format?

Reply:
so for a single record too we need a file to be created?  I think we should not change the original command..

Reply:
no for a single record deletion the old command will continue to work